cpp/libcudf_kafka/src/kafka_consumer.cpp

```diff
@@ -153,6 +153,26 @@ int64_t kafka_consumer::get_committed_offset(std::string const &topic, int parti
   return offset > 0 ? offset : 0;
 }
 
+const std::map<std::string, std::vector<int32_t>> kafka_consumer::list_topics()
+{
+  RdKafka::Metadata *md;
+  std::map<std::string, std::vector<int32_t>> topic_parts;
```

Comment:
Are we returning just a list of partition integer values? It seems like the Confluent library returns so much more.  What does the result of the call to `consumer->metadata(...)` return? Do we need to parse the results? Would it make to not do any iterations here and just have this function be a pass-through for the `consumer-metadata(...)` function?